spark-jobs/src/main/resources/reference.conf

```diff
@@ -26,7 +26,7 @@ menas.rest.uri="http://localhost:8080,host2:9000/menas;https://localhost:8080,ho
 
 # 'enceladus_record_id' with an id can be added containing either true UUID, always the same IDs (row-hash-based) or the
 # column will not be added at all. Allowed values: "uuid", "stableHashId", "none"
-enceladus.recordId.generation.strategy="uuid"
+enceladus.recordId.generation.strategy="stableHashId"
```

Comment:
I really don't think this is a good default. Happy to discuss.

Reply:
I think it will be better to have this necessity in readme of the tests.